pkg/enactment/refresh.go

```diff
@@ -0,0 +1,18 @@
+package enactment
+
+import (
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	EnactmentRefresh          = 5 * time.Hour
+	EnactmentRefreshMaxFactor = 0.1
+)
+
+// EnactmentRefreshWithJitter add some jitter to to the refresh rate so it does
+// not hit apiserver at the same time.
+func EnactmentRefreshWithJitter() time.Duration {
```

Comment:
Please consider naming this function just `EnactmentRefreshWithJitter`, that way this func would be called as `enactment.RefreshRateWithJitter()`, which reads better

Reply:
done